xdp-root/xdp_root.c

```diff
@@ -115,7 +134,7 @@ int main(int argc, char **argv)
 	    return EXIT_FAILURE;
     }
 
-    ifindex_in = if_nametoindex(iface);
```

Comment:
this didn't work so replaced it with if_stringtoindex

Reply:
What didn't work about it?  Were you using the ifName?

Reply:
I don't remember what exactly it was returning but with all fixes, this seems to be working so reverted it